templates/python/custom-copilot-assistant-assistants-api/src/config.py.tpl

```diff
@@ -15,6 +15,8 @@ class Config:
     PORT = 3978
     APP_ID = os.environ.get("BOT_ID", "")
     APP_PASSWORD = os.environ.get("BOT_PASSWORD", "")
+    APP_TYPE = os.environ.get("MicrosoftAppType", "")
```

Comment:
The environment variable naming convention only allows upper case, can we use the ENVIRONMENT_VARIABLE_NAME style as the environment variable name?

Reply:
Please also update other similar environment variable names